lib/ansible/modules/cloud/openstack/os_stack.py

```diff
@@ -229,9 +237,12 @@ def main():
                            supports_check_mode=True,
                            **module_kwargs)
 
-    # stack API introduced in 1.8.0
-    if not HAS_SHADE or (StrictVersion(shade.__version__) < StrictVersion('1.8.0')):
-        module.fail_json(msg='shade 1.8.0 or higher is required for this module')
+    # stack tag API introduced in 1.x.x
+    if not HAS_SHADE or (StrictVersion(shade.__version__)
+                         < StrictVersion('1.2.6')):
+        module.fail_json(
+            msg='shade 1.2.6 or higher is required for this module')
```

Comment:
could you please elaborate why this gets changed from 1.8.0 to 1.2.6?

Reply:
my typo, 1.26.0, I've just updated the PR